toolbox/cijoe-pkg-xnvme/testcases/examples-xnvme_hello.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
```

Comment:
The commit title should be something along the lines of: ``toolbox:cijoe: change shebang to '/usr/bin/env bash'``

And the body should describe why this is needed.